cli/src/test/scala/com/codacy/analysis/cli/ToolSelectorSpec.scala

```diff
@@ -74,40 +74,13 @@ class ToolSelectorSpec extends Specification with NoLanguageFeatures {
   }
 
   "AnalyseExecutor.tools" should {
-    "use input over remote configuration" in {
```

Comment:
This test was testing something that is impossible now that we don't pass the configuration anymore.